### ISSUE TITLE
Write regression tests against sample file from organice

### ISSUE THREAD

**Issue by munen** [200ok-ch/org-parser#1]
https://github.com/200ok-ch/organice/blob/master/sample.org

**Comment by schoettl** [comment]
Good idea. I think it would already be usefull now, even if we don't yet support everything from organice. But at least, there should be no parse failures.

**Comment by munen** [comment]
👍
